README.md

```diff
@@ -43,6 +51,125 @@ class MyParameterizedTest {
+    androidTestImplementation "io.qameta.allure:allure-kotlin-commons:$LATEST_VERSION"
+    androidTestImplementation "io.qameta.allure:allure-kotlin-junit4:$LATEST_VERSION"
+    androidTestImplementation "io.qameta.allure:allure-kotlin-android:$LATEST_VERSION"
+}
+```
+
+#### Attaching listener
+
+AndroidX Test introduced a new `AndroidJUnit4` class runner that can be used for both **Robolectric** and **on-device instrumentation tests**. The same pattern is used for `AllureAndroidJUnit4` class runner. It attaches the allure listener to current class runner, but under the hood it uses `AndroidJUnit4`. All you need to do is to add `@RunWith(AllureAndroidJUnit4::class)` annotation to your test. 
+
+```kotlin
+@RunWith(AllureAndroidJUnit4::class)
+class MyInstrumentationTest {
+    ...
+}
+```
+
+Using AllureAndroidJUnit4 over class - works for both robolectric and on-device tests.
+
+#### Robolectric tests
+
+Robolectric tests are simple unit tests, hence the API is the same. The report data will be placed in the same place as for unit tests. 
+
+#### On-device instrumentation tests
+
+You can also use testInstrumentationRunner for setting up runner.
+
+```
+android {
+    defaultConfig {
+        testInstrumentationRunner "io.qameta.allure.android.runners.AllureAndroidJUnitRunner"
+    }
+}
+```
+
+##### Integration
+As on-device instrumentation test run on an actual device, the results have to be saved there as well. To do so permissions for accessing external storage are needed. If your app doesn't have those permissions, you can include them only in your debug build type (or any other build type under which the tests are executed):
+
+**src/debug/AndroidManifest.xml**
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="io.qameta.allure.sample.junit4.android">
+
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+</manifest>
+```
+
+Moreover, Allure will grant itself those permissions at runtime, so you don't have to place any special logic for Android 6.0+ devices. 
+
+After the tests are finished you can pull the results from the external storage using an **adb** command like this one:
+```
+adb pull /sdcard/allure-results
+```
+Finally, you can generate the report via Allure CLI (see the [Allure Documentation][allure-cli]) or generate report with [allure-gradle][allure-gradle-plugin] plugin.
+
+
+##### Features
+
+The Allure Android API includes couple of features to make your reports a bit better.
+
+###### Screenshot attachment
+
+Screenshot can be taken and appended as an attachment to step or test in which they were executed:
+```kotlin
+@Test
+fun screenshotExample() {
+    step("Step screenshot") {
+        Allure.allureScreenshot(name = "ss_step", quality = 90, scale = 1.0f)
```

Comment:
minor: just `allureScreenshot` not `Allure.allureScreenshot`. It's a global function right now.